python_opp/first_hm/main.py

```diff
@@ -0,0 +1,29 @@
+class Product:
+    def __init__(self, name, price):
+        self.name = name
+        self.price = price
+    def total_price(self, quantity):
+        return round(self.price * quantity, 2)
+
+class ShoppingCart:
+    def __init__(self):
+        self.order = {}
+    def add_product(self, product: Product, quantity):
+        self.order.update({product: quantity})
+    def total_price(self):
+        result = 0
+        for (product,amount) in self.order.items():
+            result += round((product.price * amount), 2)
```

Comment:
У тебе он вище є метод для цього :-)
```suggestion
            result += product.total_price(amount)
```

Reply:
точняяк, а я думаю для цього ми створювали цей метод в класі продукт))))))))))))